com.protonvpn.www.yml

```diff
@@ -0,0 +1,513 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+app-id: com.protonvpn.www
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: protonvpn
+separate-locales: false
+finish-args:
+  - --device=dri
```

Comment:
Why was this added? This would be absolutely needed if for example the app was GTK 4, but last I checked it used GTK 3.

Reply:
Unless they do some custom gl thing, a general gtk3 wouldn't need this.

Reply:
Good question, I didn't consider that. I asked around and it indeed it is probably a good idea to have it enabled.